projects/storefrontapp-e2e-cypress/cypress/helpers/cart-coupon.ts

```diff
@@ -15,15 +16,20 @@ export const myCouponCode2 = 'midautumn';
 export function addProductToCart(productCode: string) {
   cy.get('cx-searchbox input')
     .clear()
-    .type(`${productCode}{enter}`, { force: true });
+    .type(`${productCode}{enter}{enter}`, { force: true });
```

Comment:
I was able to make it work successfully many times with :
  cy.get('cx-searchbox input')
    .clear({ force: true })
    .type(`${productCode}{enter}`, { force: true });